client/src/components/about/AboutUs.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Button, Col, Container, Row } from 'reactstrap'
+import theDire from '../../img/the_dire.jpg'
+import { createUrl } from '../../api/utils'
+
+export const Intro = styled.div`
+    margin: 2rem 0;
+    text-align: center;
+`
+const Description = styled.p`
+    max-width: 880px;
+    margin: 0 auto 30px auto;
+`
+const Divider = styled.hr`
+    margin: 5rem 0;
+`
+const MarketingImage = styled.img`
+    max-width: 400px;
+`
+const MarketingList = styled.ol`
+    margin-left: 2rem;
+`
+
+const AboutUs = props => (
+    <Container fluid>
+        <Container>
+            <Intro>
```

Comment:
I would recommend not using `<Intro>` here (or at least `text-align-center`) when you have long content like this. Left-justified (default) is usually easier to read for large sections of text.